Tests/TestCodeGenConfigurations/EmbeddedInTarget-InSchemaModule/apollo-codegen-config.json

```diff
@@ -13,7 +13,8 @@
     },
     "pruneGeneratedFiles" : true,
     "queryStringLiteralFormat" : "multiline",
-    "warningsOnDeprecatedUsage" : "include"
+    "warningsOnDeprecatedUsage" : "include",
+    "operationDocumentFormat" : ["operationId"]
```

Comment:
We don't need to address this in this current PR. But for integration testing individual configuration options, maybe we could build something better here.

We need to have these multiple different project setups in `TestCodeGenConfigurations` for testing the actual project configurations (Cocoapods, SPM, Xcode) and codegen methods (Relative vs absolute, schema module embedded, etc).

These are things that affect how/where the files are generated and if they include the correct namespacing and import statements to link and compile properly.

But for testing other individual configuration properties which would be the same regardless of these configurations, we should probably have a single project that we re-generate with different configurations so that we can isolate and test _just_ the configuration options we want to test. These could be done using the Swift Scripting codegen to make it easier to run multiple of them rapidly. 

This will take some time to set up properly, so it's not something I think we need to do right now. But this isn't the first time I've noticed that we are kind of overloading these integration tests, so it might be worth thinking about.